src/web/routes/application/manual-address/maunal-address.test.js

```diff
@@ -73,3 +73,34 @@ test('Address contentSummary() should return content summary in correct format w
   t.deepEqual(result, expected, 'should return content summary in correct format with address line 2 undefined')
   t.end()
 })
+
+test('isNavigable() returns true when there`s no session', (t) => {
```

Comment:
Is ``` the correct type of apostrophe for `there`s`? I think it should be `there’s` or `there is`.

Reply:
I thought that was only the case for user content? Either way, I've updated to 'there is'.